src/mscorlib/src/System/String.cs

```diff
@@ -770,15 +770,21 @@ unsafe public void CopyTo(int sourceIndex, char[] destination, int destinationIn
         [System.Security.SecuritySafeCritical]  // auto-generated
         unsafe public char[] ToCharArray() {
             int length = Length;
-            char[] chars = new char[length];
             if (length > 0)
             {
-                fixed (char* src = &this.m_firstChar)
-                    fixed (char* dest = chars) {
-                        wstrcpy(dest, src, length);
-                    }
+                char[] chars = new char[length];
+                fixed (char* src = &this.m_firstChar, dest = chars)
```

Comment:
No reason in particular; I'll change it back in a minute or two to multiple `fixed` statements.